### PR TITLE
Add functionalitiy "View sorting on video" and "Clickable concatenated trace "

### PR DESCRIPTION
**Pull request by mca-sh**
Here is a release note:
* [Fixed bugs](#fixed-bugs)
* [Improvements](#improvements)
* [New functionalities](#new-functionalities)

# Fixed bugs
##### Sample management
- **fix error** when using hash-tagged input color in `hex2rgb.m`
- **correct** automatic sorting when molecule selection is modified after last update (selection at last update is now saved)

# Improvements
##### Sample management
- **modify GUI** in Trace manager for new functionality "View sorting on video" 
- **make more robust** mask plot and selection by click by handling properly the axes handle and the range limits after clicking
- **add** a concatenated trace plot in tool "Automatic sorting" to visualize which trace is included/excluded from the current range.

screenshot_AS2

# New functionalities
##### Sample management
- **View sorting on video** allows to visualize molecule subgroups on the average video frame. Subgroups are defined by their respective tags or by the molecule selection.

screenshot_VV

- **Clickable concatenated trace** allows to directly click on the concatenated trace to scroll the panel "Molecule selection". Selected molecules are indicated on the concatenated trace with a mask.

screenshot_OV6

### CODE REVIEW

File: MASH-FRET/source/mod-trace-processing/management/traceManager.m [hunk 1]
```diff
@@ -2071,6 +2046,62 @@ function plotData_autoSort(h_fig)
 
 end
 
+function drawMask(h_fig,x,y,D)
+
+% defaults
+clr = [0.5,0.5,0.5];
```
Comment (fdsteffen) [inline MASH-FRET/source/mod-trace-processing/management/traceManager.m]: In the tabs "Auto sorting" all masked intensity/FRET regions are grayed out (i.e. everything not within xlow to xup). In the "Overview" tab the unselected molecules are grayed out too. This is intuitive. However in the upper panel of the "Overview" tab where the concatenated trace is displayed, the grayed area corresponds to the molecules currently in view. I suggest to do the following for consistency: Gray out the area in the concatenated trace corresponding to all "unselected" molecules and color the area of the molecules currently in view with a faint desaturated blue or similar (both in the concatenated traces and in the molecule selection panel). What do you think?

Comment (mca-sh) [reply MASH-FRET/source/mod-trace-processing/management/traceManager.m]: Yes! I got the same idea! In MASH-code 6 the molecules in view are on a white background in the concatenated traces, and the molecules out of view ar masked in grey. I don't think adding another color (blue) would bring more help, to my opinion, white is sufficient (TM has already a lot of colors)

Comment (fdsteffen) [reply MASH-FRET/source/mod-trace-processing/management/traceManager.m]: Agreed!

**Comment by fdsteffen** [comment]
@pflanzenhueter Suggesting to merge this!

**Pull request merged**